### ISSUE TITLE
After merging custumizable plots, tests segfault on Linux

### ISSUE THREAD

**Issue by markotoplak** [biolab/orange3#4896]
Started happening after #4828 was merged. Linux, Ubuntu 18.04, Python 3.7, PyQt 5.12. I have no problems with reports or help windows in Orange.

If I run `python -m unittest Orange.widgets.visualize.tests.test_owlineplot` I get a segfault with the following backtrace:

```
#0  0x00007fffc4aed970 in QOpenGLContext::isValid() const () from /home/marko/venv37/lib/python3.7/site-packages/PyQt5/Qt/lib/libQt5Gui.so.5
#1  0x00007fffb79bd8b9 in ?? () from /home/marko/venv37/lib/python3.7/site-packages/PyQt5/Qt/lib/libQt5WebEngineCore.so.5
#2  0x00007fffb79be90c in ?? () from /home/marko/venv37/lib/python3.7/site-packages/PyQt5/Qt/lib/libQt5WebEngineCore.so.5
#3  0x00007fffbe60015e in QWebEnginePagePrivate::QWebEnginePagePrivate(QWebEngineProfile*) ()
   from /home/marko/venv37/lib/python3.7/site-packages/PyQt5/Qt/lib/libQt5WebEngineWidgets.so.5
#4  0x00007fffbe600e54 in QWebEnginePage::QWebEnginePage(QObject*) ()
   from /home/marko/venv37/lib/python3.7/site-packages/PyQt5/Qt/lib/libQt5WebEngineWidgets.so.5
#5  0x00007fffbe60e02b in QWebEngineView::page() const ()
   from /home/marko/venv37/lib/python3.7/site-packages/PyQt5/Qt/lib/libQt5WebEngineWidgets.so.5
#6  0x00007fffbe835b59 in meth_QWebEngineView_page () from /home/marko/venv37/lib/python3.7/site-packages/PyQt5/QtWebEngineWidgets.so
#7  0x0000555555684e67 in _PyMethodDef_RawFastCallKeywords ()
#8  0x0000555555684c70 in _PyCFunction_FastCallKeywords ()
#9  0x00005555556f88c0 in _PyEval_EvalFrameDefault ()
#10 0x00005555556f38ce in _PyEval_EvalCodeWithName ()
```

Also some other tests segfault, but Orange seems to be behaving normally.

Happens both  with pyqtgraph 1.0.0 and 1.1.0.

Intentionally disabling WebEngine by breaking the import "fixes" segfaults.

**Comment by markotoplak** [comment]
Duplicate of #4934
